Client/src/component/NewReviewForm.jsx

```diff
@@ -0,0 +1,379 @@
+/* eslint-disable consistent-return */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+// import PropTypes from 'prop-types';
+import ReactDom from 'react-dom';
+
+const MODAL_STYLES = {
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  backgroundColor: '#FFF',
+  padding: '50px',
+  zIndex: 1000,
+};
+const OVERLAY_STYLES = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
+  backgroundColor: 'rgba(0, 0, 0, .7)',
+  zIndex: 1000,
+};
+
+const NewReviewForm = ({ showNewReviewModal, setNewReviewModal }) => {
+  const [rating, clickStar] = useState(5);
+  const [recommend, isRecomnended] = useState(undefined);
+  const [summary, changeSummary] = useState('');
```

Comment:
[summary, setChangeSummary] , put 'set'  on any setState will be more readable

Reply:
Done!